File: API/Manager/Manager/TestHistory/Models/TestHistoryUnit.cs
using Manager.Common;

namespace Manager.Manager.TestHistory.Models;

public class TestHistoryUnit
{
    public string Username { get; set; } = string.Empty;
    public List<SelectedAnswer> SelectedAnswers { get; set; } = [];
    public Score Score { get; set; }
    public string StartDate { get; set; } = string.Empty;
    public string FinishDate { get; set; } = string.Empty;
    public int TestHistoryId { get; set; }
}

File: Clients/testmanager/src/app/manager/models/types/testHistoryUnit.ts
import { Score } from '../../../common/models/types/score';
import { SelectedAnswer } from '../../../common/models/types/selectedAnswer';

export type TestHistoryUnit = {
  username: string;
  selectedAnswers: SelectedAnswer[];
  score: Score;
  startDate: string;
  finishDate: string;
  testHistoryId: number;
};
